### PR TITLE
Adding more files and fix format

### LINKED ISSUE #1: [Proposal] Databases
Thanks for this guide guys...

I'm also have a function `public func databases(config: inout DatabasesConfig)` to register the database (or databases), now i have this function in the `migrate` file because are related, but also can be in his own file. I think should be a nice addition to this guide.

### PR DESCRIPTION
**Pull request by gperdomor**
Format fixed by VSCode

Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,31 +106,44 @@ The final application structure (inside the Sources folder) looks like this:
...
 
 **Note**: For more information on registering credentials and secrets, see the “Credentials” section.
 
 ### Files
+
 There are 6 files you should have:
 
 - app.swift (use the default template version)
 - boot.swift (use the default template version)
+- commands.swift // Optional
```
Comment (jdmcd) [inline README.md]: Tiny detail - can we use parenthesis here instead of a comment-like syntax?

File: README.md [hunk 2]
```diff
@@ -91,31 +106,44 @@ The final application structure (inside the Sources folder) looks like this:
...
 - configure.swift
-- migrate.swift
+- content.swift
+- databases.swift // Optional
```
Comment (jdmcd) [inline README.md]: Here as well (in addition to migrate and repositories)

File: README.md [hunk 3]
```diff
@@ -136,8 +164,121 @@ You should call this function from `configure.swift` like this:
 
 For more information on routes, see the “Routes and Controllers” section.
 
+### commands.swift
+
+Use this file to add your custom commands to your application. Example:
```
Comment (jdmcd) [inline README.md]: `Example` -> `For example:` please

**Review by jdmcd — CHANGES_REQUESTED** [review]
Overall looks great, just a few small comments. Thanks for this great addition!

File: README.md (hunk 1 above)
Comment (gperdomor) [reply README.md]: of course... my bad

Comment (jdmcd) [reply README.md]: No worries! :)

**Review by jdmcd — APPROVED** [review]

**Comment by jdmcd** [comment]
@gperdomor Thanks again!

**Pull request merged**